HW06.Rmd

```diff
@@ -45,44 +73,80 @@ ens.mapped_GBM <- left_join(ens.GBM.transcripts, map, by = c("gene_id" = "TXID")
   dplyr::select(-1) %>% 
   dplyr::select(gene_symbol = SYMBOL, everything())
 
-ens.mapped_GBM <- ens.mapped_GBM[!duplicated(ens.mapped_GBM$gene_symbol),] #remove duplicated gene symbols
+#Since I prepared my data frame long-wise and I intended to have duplicated gene symbols (each duplicate has their own TPM values assigned), I'm commenting out the line below
+

```

Comment:
I also prepared my data frame long-wise and wasn't sure how to alter Robert's code to make it work for mine. So it's helpful to see how someone else addressed this!